presto-docs/src/main/sphinx/release/release-312.rst

```diff
@@ -0,0 +1,67 @@
+===========
+Release 312
+===========
+
+General Changes
+---------------
+
+* Fix incorrect results for queries using ``IS [NOT] DISTINCT FROM``. (:issue:`795`)
+* Fix ``array_distinct``, ``array_intersect`` semantics with respect to indeterminate
+  values (i.e., ``NULL`` or structural types containing ``NULL``). (:issue:`559`)
+* Fix failure when the largest negative ``BIGINT`` value (-9223372036854775808) is used as a
+  constant in a query. (:issue:`805`)
+* Improve reliability for network errors when using Kerberos with
+  :doc:`/security/internal-communication`. (:issue:`838`)
+* Improve performance of ``JOIN`` queries involving inline tables (``VALUES``). (:issue:`743`)
+* Improve performance of queries containing duplicate expressions. (:issue:`730`)
+* Improve performance of queries involving comparisons between values of different types. (:issue:`731`)
+* Improve performance of queries containing redundant ``ORDER BY`` clauses in subqueries. This may
+  affect the semantics of queries that incorrectly rely on implementation-specific behavior. The
+  old behavior can be restored via the ``skip_redundant_sort`` session property or the
+  ``optimizer.skip-redundant-sort`` configuration option. (:issue:`818`)
+* Improve performance of ``IN`` predicate containing a subquery. (:issue:`767`)
+* Improve support for corrected subqueries containing redundant ``LIMIT`` clauses. (:issue:`441`)
+* Remove deprecated check for minimum number of workers before starting a coordinator.  Use the
+  ``query-manager.required-workers` and ``query-manager.required-workers-max-wait`` configurations
+  instead. (:issue:`95`)
+* Raise required Java version to 8u161. This version allows unlimited strength crypto. (:issue:`779`)
```

Comment:
This seems import, and the list of changes is long. Maybe we should move this and the JVM changes to a "Server Changes" section.

Reply:
Would the TrustStore-related change go there, too? How about the entry about minimum number of workers config property?

Reply:
Yeah, that seems like a good idea